sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/DateTimeUtilsSuite.scala

```diff
@@ -820,4 +820,9 @@ class DateTimeUtilsSuite extends SparkFunSuite with Matchers with SQLHelper {
       }
     }
   }
+
+  test("SPARK-35679: handled overflow error raised by instantToMicros") {
```

Comment:
SPARK-35679: instantToMicros should be able to return microseconds of Long.MinValue

Reply:
thanks